### DISCUSSION TITLE
Gigabyte A520l AC, Ryzen 7 4700G, 16 GB VRAM

### DISCUSSION THREAD

**Discussion by SalmonPete** [ChefKissInc/NootedRed#171] [Show and tell]
Also runs Stable Diffusion in macOS Sonoma!. A 512x512 image in 40 steps takes 1 min 27 sec. A 1024x1024 image in 40 steps takes about 9 minutes.

![image](https://github.com/ChefKissInc/NootedRed/assets/143473778/a6b448c3-e017-48ca-bc59-371dd16b6fb7)
![Screenshot 2023-09-13 at 12 16 48 AM](https://github.com/ChefKissInc/NootedRed/assets/143473778/c2bb9599-6066-4c1f-89ea-7519d46bdf09)
![Screenshot 2023-09-13 at 12 35 31 AM](https://github.com/ChefKissInc/NootedRed/assets/143473778/72066660-a77c-47dc-bd20-a96473be7b6b)

![image](https://github.com/ChefKissInc/NootedRed/assets/143473778/66c467f0-5f78-4957-96c9-11f0591bf2c3)

![image](https://github.com/ChefKissInc/NootedRed/assets/143473778/2bbd1710-8dad-47bd-913b-009c94ac1dfc)

![image](https://github.com/ChefKissInc/NootedRed/assets/143473778/22553a25-c72c-4fda-8bfa-9f362b96ee28)

**Comment by SalmonPete** [comment]
Latest NootedRed kext have really boosted the performance of the 4700G. 1st Screen shot is dated December, the 2nd yesterday. I have benched more recently but haven't seen a boost this noticeable before!. Great work guys, thanks.


![Screenshot 2023-12-03 at 1 55 33 PM](https://github.com/ChefKissInc/NootedRed/assets/143473778/7fdcfa28-9a68-43a1-82c0-e523f5744428)
![Screenshot 2024-03-15 at 1 11 02 AM](https://github.com/ChefKissInc/NootedRed/assets/143473778/f7a7781f-2e44-469d-9847-9797b961ee9f)

**Comment by SalmonPete** [comment]
Successfully updated to macOS Sonoma 14.4.1. The desktop image was created using Stable Diffusion on the 4700G.

![Screenshot 2024-03-29 at 10 07 59 PM](https://github.com/ChefKissInc/NootedRed/assets/143473778/03194493-04b2-409e-bcd9-f6fcbb167afc)

**Comment by SalmonPete** [comment]
Amazing work @VisualEhrmanntraut thank you.

![Screenshot 2024-06-15 at 11 40 59 PM](https://github.com/ChefKissInc/NootedRed/assets/143473778/912c71ab-73b5-4902-8c24-c68d78efa936)

First task was to update my kexts folder;

- AppleALC 1.9.0 > 1.9.1
- lilu 1.6.7 > 1.6.8
- RestrictEvents 1.1.3 > 1.1.4
- VirtualSMC 1.3.2 > 1.3.3
- NootedRed, [commit e326a5f]

Disabled the following kexts;

- Airportitlwm
- AMDRyzenCPUPowerManagement
- BlueToolFixup
- IntelBluetoothFirmware
- IntelBTPatcher
- NootedRed, (I can still run Sequoia with no graphics acceleration, installation is much quicker if disabled).

Misc;
- Updated AMD Vanilla Patches
- Added ForgedInvariant.kext, a CPU TSC kext for AMD and Intel by ChefKiss. 
- Added '-lilubetaall' to my boot args [Not needed if updated lilu 1.6.8]
- Changed SecureBoot to disabled

**Reply by SalmonPete** [reply]
Latest macOS Sequoia Beta 2 working great.
![image](https://github.com/ChefKissInc/NootedRed/assets/143473778/2af17337-13eb-4637-895b-168c4097573c)

**Comment by SalmonPete** [comment]
Sequoia Beta 3 working fine with the latest NootedRed.kext and BetterDisplay 2K scaled Resolution.

![Screenshot 2024-07-10 at 11 11 02 PM](https://github.com/ChefKissInc/NootedRed/assets/143473778/312f2c3a-fda8-4b9f-a7f2-2d010d5a70fa)

**Comment by SalmonPete** [comment]
Latest macOS Public Beta working fine on the 4700G.

![Screenshot 2024-08-17 at 7 48 05 AM](https://github.com/user-attachments/assets/f8bb3701-2b78-4244-a22a-2ea5da2b332a)

**Comment by SalmonPete** [comment]
macOS Sequoia is running sweet with NootedRed, even DiffusionBee is fully working!. Thank you @VisualEhrmanntraut and the team.

![Screenshot 2024-09-18 at 10 14 20 AM](https://github.com/user-attachments/assets/bb8a9bb8-bd91-4bab-8e3a-4139257da8a4)

![DB](https://github.com/user-attachments/assets/930840b4-f881-4b2a-91d5-5b2d2cb9fa2d)

**Reply by VisualEhrmanntraut** [reply]
there's no team behind NRed, it's just me :-)
